packages/parcel-transformer/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import { Transformer } from '@parcel/plugin';
+import { compile, processVanillaFile } from '@vanilla-extract/integration';
+
+export default new Transformer({
+  async transform({ asset, options }) {
+    const identOption = options.mode === 'development' ? 'debug' : 'short';
```

Comment:
Should consumers also be able to explicitly set this via an `identifiers` option?